app/shop/data.py

```diff
@@ -1,9 +1,12 @@
+# You need `pandas` installed to
```

Comment:
to what?

Reply:
Also I vendored this in - we probably don't want to be running the tests and I think they use libraries we don't install.

Reply:
As per discussion, we'll remove this file and rely on the fixture

Reply:
Cheers for the spot